src/modules/directory.rs

```diff
@@ -39,6 +39,8 @@ pub fn segment(context: &Context) -> Option<Module> {
     let truncated_dir_string = truncate(dir_string, DIR_TRUNCATION_LENGTH);
     module.new_segment("path", truncated_dir_string);
 
+    module.get_prefix().set_value("");
```

Comment:
@matchai This change causes the directory.rs tests to fail. was the `via` prefix needed, or was it just the default?

Reply:
`via ` was the default, but I believe it should be `in `.
We may need to add some logic to skip the prefix of the first segment that isn't `None`. At the moment, I believe it skips the first prefix, regardless of whether it's a `None` or a `Some`.

Reply:
Looking it over, it seems [we strip out `None` values before skipping the first prefix](https://github.com/starship/starship/blob/master/src/print.rs#L38-L46), so no additional work should be needed here.

Reply:
Just to clarify, what I meant is that the `directory` segment defaults to the `via ` prefix and causes this to happen:
![Screenshot from 2019-05-16 06-16-29](https://user-images.githubusercontent.com/1578005/57830679-5ff7e680-77a2-11e9-8e7d-c00299bc6279.png)
So, should I keep the change I made and have the `username` segment add the `in ` suffix, or should the `directory` segment be the one adding the prefix `in `? which one would make more sense?

Reply:
Directory should be the one with the `in ` prefix since it would make sense with other prompt orders and configurations.